storybookjs/storybook
# Add deprecation warnings when addons are required via main package

Issue: #1017

## What I did

Use `action` and `linkTo` addons just like any other storybook addon.
Update examples, docs, and add deprecation warning to code.

## How to test
First run:
```
npm run bootstrap
cd examples/cra-storybook
npm run storybook
```
Open browser console, and there should be no deprecation warnings.
Then, update the stories to use:

```js
import { storiesOf, action, linkTo from } from '@kadira/storybook'
```
Deprecation warnings should appear in the browser console.
